changelog/18.0/18.0.0/summary.md

```diff
@@ -118,6 +119,35 @@ VTGate flag:
 
 - `--schema_change_signal_user` is deprecated and will be removed in `v19.0`
 
+#### <a id="deleted-flags"/>Deleted Command Line Flags
+
+Flags in `vtcombo`:
+- `--vtctld_addr`
+
+Flags in `vtctldclient ApplySchema`:
+- `--skip-preflight`
+
+Flags in `vtgate`:
+- `--vtctld_addr`
+
+Flags in `vttablet`:
+- `--vtctld_addr`
+- `--use_super_read_only`
+- `--disable-replication-manager`
+- `--init_populate_metadata`
```

Comment:
FWIW we said that we *deleted* this flag in the v16 summary. Maybe we meant deprecated there? Or maybe that's the one that we had to temporarily resurrect... 😄

Reply:
Originally it was meant to be `deprecated` in `v16.0.0`: https://github.com/vitessio/vitess/commit/4f5ab229213de475cb4d46f524f566ebf89167fa

Reply:
and went from `deprecated` to `removed` in the release notes via: https://github.com/vitessio/vitess/commit/ebfe6fe611e241b9047df184e7060b84a2019fe2

Reply:
but the flag has always been marked as deprecated, so all good. I can change the `v16.0.0` release notes if you think it would be useful.